components/Achievements/Achievements.module.scss

```diff
@@ -0,0 +1,151 @@
+    align-items: center;
+    padding-right: 0px;
+    .btn {
+      align-self: center;
+      button {
+        background-color: #3b7a9e;
+        text-decoration: none;
+        transition: all 0.2s;
+        border: none;
+      }
+      button:hover {
+        cursor: pointer;
+      }
+    }
+    .slider {
+      max-width: 100vw;
+      width: 100%;
+      overflow: hidden;
+      .group3646 {
+        background-color: #3b7a9e;
+        display: flex !important;
+        @media (max-width: 899px) {
+          flex-direction: column;
+        }
+        @media (min-width: 900px) {
+          flex-direction: row;
+        }
+        justify-content: space-around;
+        align-items: center;
+        flex-wrap: wrap;
+        .achievementinfo {
+          display: flex !important;
+          flex-direction: column;
+          align-items: flex-start;
+          flex: 1;
+          margin: 1rem;
+          margin-left: 1.5rem;
+          padding: 0.1rem;
+          justify-content: space-between;
+          color: #f9f6ff;
+          .year {
+            text-transform: uppercase;
+            background: #f9f6ff;
+            border-radius: 40px;
+            padding: 0.1rem 2rem;
+            text-align: center;
+            font-family: 'Manrope';
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1rem;
+            letter-spacing: 0.05em;
+            color: #3b7a9e;
+          }
+          .title {
+            margin-top: 1.5rem;
+            text-transform: uppercase;
+            height: auto;
+            font-family: 'Manrope';
+            font-style: normal;
+            font-weight: 700;
+            font-size: 2.2rem;
+            word-break: keep-all;
+            @media (max-width: 1050px) {
+              line-height: 1.6em;
+            }
+            @media (min-width: 1050px) {
+              line-height: 2em;
+            }
+            letter-spacing: 0.05em;
+            color: #f9f6ff;
+          }
+          .desc {
+            margin-top: 2.6rem;
+            word-break: keep-all;
+            font-family: 'Manrope';
+            font-style: normal;
+            font-weight: 400;
+            font-size: 1.4rem;
+            @media (max-width: 1050px) {
+              line-height: 1.6em;
+            }
+            @media (min-width: 1050px) {
+              line-height: 2em;
+            }
+            color: #f9f6ff;
+          }
+          .descEllipsis {
+            @media (max-width: 1050px) {
+              text-overflow: ellipsis;
+              overflow: hidden;
+              display: -webkit-box;
+              -webkit-line-clamp: 5;
+              -webkit-box-orient: vertical;
+            }
+          }
+        }
+        .image120 {
+          @media (max-width: 899px) {
+            margin: 0.2rem;
+            padding: 1rem 0rem;
+          }
+          @media (min-width: 900px) {
+            padding: 7rem 1rem 7rem 5rem;
+          }
+          flex-grow: 2;
```

Comment:
Why are we using so many flex-specific values? Can we do everything in just flex: `____` instead?